api/src/main/resources/workbench-api.yaml

```diff
@@ -5178,9 +5180,12 @@ definitions:
         format: int64
       username:
         type: string
+      accessTierShortNames:
+        type: string
+        description: Short names of access tiers available to the user, comma-delimited if multiple.
       dataAccessLevel:
```

Comment:
Don't remove these fields from the entity model, to allow for a transition period.  They'll be removed in RW-6189.